packages/block-editor/src/components/block-mover/index.js

```diff
@@ -44,13 +52,17 @@ function BlockMover( {
 				normalizedClientIds[ normalizedClientIds.length - 1 ]
 			);
 			const blockOrder = getBlockOrder( _rootClientId );
+			const { layout = {} } = getBlockAttributes( _rootClientId ) ?? {};
 
 			return {
 				canMove: canMoveBlocks( clientIds, _rootClientId ),
 				rootClientId: _rootClientId,
 				isFirst: firstIndex === 0,
 				isLast: lastIndex === blockOrder.length - 1,
 				orientation: getBlockListSettings( _rootClientId )?.orientation,
+				// TODO: Doesn't feel great to couple BlockMover and grid layouts.
+				// TODO: Can we use useLayout() instead?
+				isManualGrid: layout.type === 'grid' && !! layout.columnCount,
```

Comment:
Can we disable block movers here without referring to layout at all?

Reply:
Yeah, I can't really think of an alternative approach. 

Maybe if we had a more flexible way of disabling moving than the existing locking APIs. Then we could have the `layout` block-supports call an API to lock moving. But I don't think it's worth adding that just for this feature.

Should we use `useLayout` though instead of reading the attributes directly? I can't remember what benefit `useLayout` provides, if any.

Reply:
`useLayout` doesn't return the correct layout type here; it only ever returns "default" in my testing. I don't think this component has access to the context from `BlockListItems`, which is what `useLayout` returns.
If it did work, the benefit would be accessing the parent layout type through context instead of a selector.